src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.User;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
```

Comment:
Checklist: You can return information about tickets in your ShoppingCartResponseDto / OrderResponseDto in the form of List<Long> tiketIds.
Or you can create some TicketResponseDto